app/portainer/home/EnvironmentList/KubeconfigPrompt.tsx

```diff
@@ -0,0 +1,151 @@
+import { useState } from 'react';
+import { useQuery } from 'react-query';
+
+import * as kcService from '@/kubernetes/services/kubeconfig.service';
+import * as notifications from '@/portainer/services/notifications';
+import { Button } from '@/portainer/components/Button';
+import { Checkbox } from '@/portainer/components/form-components/Checkbox';
+import { EnvironmentId } from '@/portainer/environments/types';
+import { EnvironmentsQueryParams } from '@/portainer/environments/environment.service/index';
+import { isKubernetesEnvironment } from '@/portainer/environments/utils';
+import { PaginationControls } from '@/portainer/components/pagination-controls';
+import { usePaginationLimitState } from '@/portainer/hooks/usePaginationLimitState';
+import { useEnvironmentList } from '@/portainer/environments/queries/useEnvironmentList';
+
+import styles from './KubeconfigPrompt.module.css';
+import '@reach/dialog/styles.css';
+
+function useSelection() {
```

Comment:
move to another file

Reply:
Need confirm: move the definition of useSelection() to another file?
Finally, there are 3 files. button.tsx, prompt.tsx, useSeletion.tsx ?

Reply:
yes

Reply:
Done